docs/reference/commandline/daemon.md

```diff
@@ -768,6 +768,10 @@ the host, file ownership will have to be pre-arranged if the user or
 administrator wishes the containers to have expected access to the volume
 contents.
 
+To enable those advanced features for a specific container use ```-userns=host```
```

Comment:
also;
- can you replace triple back-ticks with single ones if you use inline?
- `-userns=host` has a single dash, should be `--userns=host`